test/utils/path.spec.js

```diff
@@ -1,8 +1,48 @@
 import {expect} from 'chai'
-import {parse} from '../../app/scripts/utils/path'
+import Path, {parse} from '../../app/scripts/utils/path'
```

Comment:
It's variable destructing from the module, allows you to pull named exports out.

```javascript
// in path.js
export function parse() {}
```

Which is like doing `exports.parse = parse`